csrc/python_frontend/python_bindings.cpp

```diff
@@ -218,6 +221,32 @@ struct DimInfo {
   }
 };
 
+// Return if we can schedule FusionDefinition with heuristic along with any
+// debug messages from canScheduleRejectReason.
+std::tuple<bool, std::string> canSchedule(
+    UserSchedule* sched,
+    const ScheduleHeuristic& heuristic) {
+  NVF_ERROR(
+      sched->schedule != nullptr,
+      "Requires Fusion to use heuristic schedulers");
+  NVF_ERROR(
+      sched->runtime_info != nullptr,
+      "Requires SchedulerRuntimeInfo to use heuristic schedulers");
+
+  // Enable collection of messages from canScheduleRejectReason
+  DebugDumpOptionsGuard debug_dump_options_guard;
+  DebugDumpOptionsGuard::getCurOptions().set(
+      DebugDumpOption::FusionSegmenterLog);
```

Comment:
Is this a debug thing that we forgot to remove?

Reply:
NVM, realized that FusionSegmenterLog is actually just a bad name for the scheduler rejection.